src/main/java/org/black_ixx/bossshop/misc/ClassAndVerTools.java

```diff
@@ -0,0 +1,21 @@
+package org.black_ixx.bossshop.misc;
+
+import org.bukkit.Bukkit;
+
+public class ClassAndVerTools {
+    public static boolean classExists(String className) {
+        try {
+            Class.forName(className);
+            return true;
+        } catch (ClassNotFoundException e) {
+            return false;
+        }
+    }
+
+    public static boolean VerIsGreaterThanOrEqualTo(int i){
```

Comment:
Please use Java naming conventions (camelCase) for methods. It is also shorter to use `isAtLeast`